src/components/picker/PickerItem.js

```diff
@@ -18,57 +20,58 @@ import Text from '../text';
  */
 const PickerItem = props => {
   const {
-    renderItem,
     value,
     label,
-    onPress,
     disabled,
-    isSelected,
     selectedIcon = Assets.icons.check,
     selectedIconColor = Colors.primary,
     testID
   } = props;
+  const context = useContext(PickerContext);
+  const {renderItem} = context;
+  const isSelected = isItemSelected(value, context.value);
+  const itemLabel = getItemLabel(label, value, props.getItemLabel || context.getItemLabel);
+  const accessibilityProps = {
+    accessibilityState: isSelected ? {selected: true} : undefined,
+    accessibilityHint: 'Double click to select this suggestion',
+    ...Modifiers.extractAccessibilityProps(props)
+  };
 
   useEffect(() => {
     if (_.isPlainObject(value)) {
       console.warn('UILib Picker.Item will stop supporting passing object as value & label (e.g {value, label}) in the next major version. Please pass separate label and value props');
```

Comment:
Why not use LogService instead of console?

Reply:
Fixed